appstore/appstore/urls.py

```diff
@@ -9,7 +9,12 @@
 from core import views as app_core_views
 from django_saml2_auth import views as saml2_auth_views
 
+
+
 admin.autodiscover()
+handler404 = app_core_views.handler404
```

Comment:
I know this has merged, but I was just wondering if this is a reference to [`handler404`](https://docs.djangoproject.com/en/3.2/ref/urls/#handler404)?

Reply:
Yes, why?

Reply:
Just verifying understanding thanks for the follow up!